sdk/historian/src/main/java/org/dsa/iot/historian/database/Watch.java

```diff
@@ -45,6 +45,13 @@
     private Value lastValue;
 
     public WatchUpdate getLastWatchUpdate() {
+        if (lastWatchUpdate == null) {
+            Value value = node.getValue();
+            if (value != null) {
+                SubscriptionValue subscriptionValue = new SubscriptionValue(path, value, null, null, null, null);
```

Comment:
@kaendfinger Was wondering if doing this would get my in trouble. I needed get a `SubscriptionValue` for a value, and I don't have a handle onto the DSLink's `NodeManager`. I'm a bit concerned about the hard coded null values here may break something later on, but I'm not even sure they're used somewhere else.

Here's what's the constructor looks like

```java
public class SubscriptionValue {

    private final String path;
    private final Value value;
    private final Number count;
    private final Number sum;
    private final Number min;
    private final Number max;

    public SubscriptionValue(String path, Value value,
                                Number count, Number sum,
                                Number min, Number max) {
        this.path = path;
        this.value = value;
        this.count = count;
        this.sum = sum;
        this.min = min;
        this.max = max;
    }
```